src/FSharp.Analyzers.SDK/FSharp.Analyzers.SDK.fsproj

```diff
@@ -24,5 +24,6 @@
     <PackageReference Include="FSharp.Compiler.Service" />
     <PackageReference Include="McMaster.NETCore.Plugins" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" />
+    <PackageReference Include="Glob" />
```

Comment:
Do we really want this to be inside the SDK?

Reply:
It's needed for the new globbing support of analyzers include/exclude lists.
Do you see a potential issue with it?

Reply:
I was thinking of having that only in the cli tool.
Avoid as much as dependencies as possible for the analyzer authors.

Reply:
Okay, I went with a predicate parameter. That way, editor clients can still use globing if they want.